BrowserKit/Sources/Common/Extensions/URLExtension.swift

```diff
@@ -80,6 +80,16 @@ extension URL {
         return normalizedHost.flatMap { $0 + self.path }
     }
 
+    public static func getSubdomainAndHost(from urlString: String) -> (subdomain: String?, normalizedHost: String) {
+        let normalizedHost = URL(string: urlString)?.normalizedHost ?? urlString
+        let components = normalizedHost.split(separator: ".")
```

Comment:
We should use `URLComponents` here instead.

Reply:
We already use components in the `normalizedHost` computed property, where we extract the host. Since we don't have a native method to extract the subdomain, this is why I split the string. If there are more than two components, that means we have a subdomain.

Reply:
Yes, but even the comment in `normalizedHost` mentions to use components instead.